gc/base/standard/Scavenger.cpp

```diff
@@ -4155,6 +4157,8 @@ MM_Scavenger::completeBackOut(MM_EnvironmentStandard *env)
 void
 MM_Scavenger::mainThreadGarbageCollect(MM_EnvironmentBase *envBase, MM_AllocateDescription *allocDescription, bool initMarkMap, bool rebuildMarkBits)
 {
+	uint64_t startTime;
```

Comment:
Let's make these more explicit: "scavengeStart/EndTime". I think it's more clear this way, even though it can be inferred